src/modules/movieList.js

```diff
@@ -0,0 +1,24 @@
+const displayMovies = (data) => {
+  const displayMovies = document.querySelector('#movie-list');
+  data.forEach((movie) => {
+    displayMovies.innerHTML += `<article class="movie">
+<img src="${movie.image.medium}" alt="${movie.name}">
+<h2>${movie.name}</h2>
+<p class="genre">${movie.genres}</p>
+<div class="button">
+<button>Comments</button>
+<button>Reservations</button>
```

Comment:
@Kwentsir you are doing a great job for the team! 🥇  There is no problem until now. It is optional but you can change the indention of the Template Strings to increase the readability of the code.
```suggestion
<img src="${movie.image.medium}" alt="${movie.name}">
<h2>${movie.name}</h2>
<p class="genre">${movie.genres}</p>
<div class="button">
<button>Comments</button>
<button>Reservations</button>
```
You can merge now! 👍